ValidationLibrary/Rules/HasNewestPtcsJenkinsLibRule.cs

```diff
@@ -42,7 +42,9 @@ public HasNewestPtcsJenkinsLibRule(ILogger<HasNewestPtcsJenkinsLibRule> logger,
         public async Task Init(IGitHubClient ghClient)
         {
             var versionFetcher = new ReleaseVersionFetcher(ghClient, "protacon", LibraryName);
-            _expectedVersion = await versionFetcher.GetLatest().ConfigureAwait(false);
+            var release = await versionFetcher.GetLatest().ConfigureAwait(false);
```

Comment:
ConfigureAwait(false) are basically needed only when developing re-usable libraries or is there something in function apps that enforces this?

Reply:
I don't think so, but I'm trying to keep `ValidationLibrary`  re-usable to a some degree at least. (currently it's only used by Azure Functions app and the Console Runner.